src/components/informationCard/InformationCard.css

```diff
@@ -0,0 +1,32 @@
+.infoCard {
```

Comment:
We need it in kebab-case! 🍗 - 🍗 -🍗

Reply:
Best lastig om de juiste keus te maken in naming conventions merk ik! Dus klopt het dat je voor functienamen en components gebruik maakt van Upper- of lowerCamelCase en voor class names altijd kebab case? Of is dit allebei oke, maar heeft dit voor sommigen de voorkeur?